_delphi_utils_python/tests/test_covidcast_wrapper.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime, timedelta
+from pathlib import Path
+import pandas as pd
+from delphi_utils import covidcast_wrapper
+import covidcast
+from freezegun import freeze_time
+from delphi_epidata import Epidata
+from pandas.testing import assert_frame_equal
+
+TEST_DIR = Path(__file__).parent
```

Comment:
I don't think these tests should be permanent because (a) they will slow down our tests with expensive network calls and (b) they will not give us new information once finalized (unless covidcast_wrapper code changes, which I expect to be rare). I propose that we think of this as more of a testing script that we use while iterating on this PR; once we convince ourselves that the PR code is correct, we disable it. A simple way to do this is to add this skip decorator

```suggestion
TEST_DIR = Path(__file__).parent
import pytest
# Uncomment this to run these tests, leave commented once done.
# @pytest.mark.skip(reason="Expensive external requests to verify covidcast_wrapper.")
```

Reply:
TBQH, i dont think the covidcast wrapper is necessary -- in fact, i think its overkill.  getting rid of it entirely will negate any need for this test.

Reply:
Getting rid of the wrapper seems possible, but I think these tests are still useful for ensuring correctness of this migration. Once we trust the migration, I definitely think we should disable/delete them, though having a test record is nice, whether that takes the form of a link to a commit hash or a disabled unit test, doesn't matter to me.